selecta

```diff
@@ -276,6 +266,15 @@ class Search
     end
   end
 
+  def move(direction)
```

Comment:
Or maybe remove those completely, as the ``move_cursor(..)`` now handles everything?